elements/lisk-chain/test/unit/transactions.spec.ts

```diff
@@ -128,10 +128,12 @@ describe('blocks/transactions', () => {
 					}) as TransactionJSON,
 				);
 				const notAllowedTx = chainInstance.deserializeTransaction(
-					registerSecondPassphrase({
-						passphrase: genesisAccount.passphrase,
-						secondPassphrase: 'second-passphrase',
+					registerMultisignature({
```

Comment:
better to use other transactions because multi signature will change, and same for below